Kernel/Net/IPv4.h

```diff
@@ -88,7 +112,7 @@ class [[gnu::packed]] IPv4Packet
     u8 m_dscp_and_ecn { 0 };
     NetworkOrdered<u16> m_length;
     NetworkOrdered<u16> m_ident;
-    NetworkOrdered<u16> m_flags_and_fragment;
+    NetworkOrdered<u16> m_flags_and_fragment { 0 };
```

Comment:
`NetworkOrdered<T>` is default-initialized to 0, no need to explicitly initialize it :)